torch/utils/_pytree.py

```diff
@@ -0,0 +1,150 @@
+from typing import NamedTuple, Callable, Any, Tuple, List, Dict
```

Comment:
This looks pretty well typed. Consider adding it to the mypy-strict.ini list so we keep it well typed!